### PR TITLE
docs(service): add a course to develop an attestation service

### PR DESCRIPTION
**Pull request by KaoImin**
Add a course to develop an attestation service

### CODE REVIEW

File: attestation.md [hunk 1]
```diff
@@ -0,0 +1,273 @@
...
+
+```shell
+./asset
+├── Cargo.toml
+├── rust-toolchain
+└── src
+    ├── lib.rs
+    └── types.rs
+
+./dex
```
Comment (zhouyun-zoe) [inline attestation.md:104]: `./dex` ?

File: attestation.md [hunk 2]
```diff
@@ -0,0 +1,273 @@
...
+            "attestation".to_owned()
+        ]
+    }
+}
+```
+
+到这里，所有的开发工作就完成了，运行 `cargo run` 编译并启动 dex 链。
+
+通过浏览器打开 http://localhost:8000/graphiql ，即可与 dex 链进行交互，graphiql 的使用方法参见[文档](graphql_api.md)。
+
```
Comment (zhouyun-zoe) [inline attestation.md:271]: 后面如果能加上 interact 的一个例子就更好了

**Comment by KaoImin** [comment]
ping @zhouyun-zoe

**Comment by KaoImin** [comment]
ping @zhouyun-zoe

**Pull request merged**